modules/openapi-generator/src/main/resources/kotlin-client/data_class.mustache

```diff
@@ -1,69 +1,67 @@
 {{#jvm}}
-{{#gson}}
-import com.google.gson.annotations.SerializedName
-{{/gson}}
-{{#moshi}}
-import com.squareup.moshi.Json
-{{/moshi}}
-{{#parcelizeModels}}
-import android.os.Parcelable
-import kotlinx.android.parcel.Parcelize
+	{{#gson}}
+		import com.google.gson.annotations.SerializedName
+	{{/gson}}
+	{{#moshi}}
+		import com.squareup.moshi.Json
+	{{/moshi}}
+	{{#parcelizeModels}}
+		import android.os.Parcelable
+		import kotlinx.android.parcel.Parcelize
```

Comment:
@janweinschenker did you use an IDE to reformat the mustache templates?

Reply:
yes, sorry. Reformatting was not intended. I could revert to the original formatting. Should I?

Reply:
No need to sorry. Things happened.

Please undo the formatting changes to the templates

Reply:
will do

Reply:
it's done